packages/webview_flutter/webview_flutter_platform_interface/lib/v4/src/webview_controller_delegate.dart

```diff
@@ -0,0 +1,291 @@
+    String? baseUrl,
+  }) {
+    throw UnimplementedError(
+        'loadHtmlString is not implemented on the current platform');
+  }
+
+  /// Makes a specific HTTP request ands loads the response in the webview.
+  ///
+  /// [WebViewRequest.method] must be one of the supported HTTP methods
+  /// in [WebViewRequestMethod].
+  ///
+  /// If [WebViewRequest.headers] is not empty, its key-value pairs will be
+  /// added as the headers for the request.
+  ///
+  /// If [WebViewRequest.body] is not null, it will be added as the body
+  /// for the request.
+  ///
+  /// Throws an ArgumentError if [WebViewRequest.uri] has empty scheme.
+  Future<void> loadRequest(
+    LoadRequestParams params,
+  ) {
+    throw UnimplementedError(
+        'loadRequest is not implemented on the current platform');
+  }
+
+  /// Accessor to the current URL that the WebView is displaying.
+  ///
+  /// If no URL was ever loaded, returns `null`.
+  Future<String?> currentUrl() {
+    throw UnimplementedError(
+        'currentUrl is not implemented on the current platform');
+  }
+
+  /// Checks whether there's a back history item.
+  Future<bool> canGoBack() {
+    throw UnimplementedError(
+        'canGoBack is not implemented on the current platform');
+  }
+
+  /// Checks whether there's a forward history item.
+  Future<bool> canGoForward() {
+    throw UnimplementedError(
+        'canGoForward is not implemented on the current platform');
+  }
+
+  /// Goes back in the history of this WebView.
+  ///
+  /// If there is no back history item this is a no-op.
+  Future<void> goBack() {
+    throw UnimplementedError(
+        'goBack is not implemented on the current platform');
+  }
+
+  /// Goes forward in the history of this WebView.
+  ///
+  /// If there is no forward history item this is a no-op.
+  Future<void> goForward() {
+    throw UnimplementedError(
+        'goForward is not implemented on the current platform');
+  }
+
+  /// Reloads the current URL.
+  Future<void> reload() {
+    throw UnimplementedError(
+        'reload is not implemented on the current platform');
+  }
+
+  /// Clears all caches used by the [WebView].
+  ///
+  /// The following caches are cleared:
+  ///	1. Browser HTTP Cache.
+  ///	2. [Cache API](https://developers.google.com/web/fundamentals/instant-and-offline/web-storage/cache-api) caches.
+  ///    These are not yet supported in iOS WkWebView. Service workers tend to use this cache.
+  ///	3. Application cache.
+  Future<void> clearCache() {
+    throw UnimplementedError(
+        'clearCache is not implemented on the current platform');
+  }
+
+  /// Clears the local storage used by the [WebView].
+  Future<void> clearLocalStorage() {
+    throw UnimplementedError(
+        'clearLocalStorage is not implemented on the current platform');
+  }
+
+  /// Sets the [NavigationCallbackDelegate] containing the callback methods that
+  /// are called during navigation events.
+  Future<void> setNavigationCallbackDelegate(
```

Comment:
`setNavigationCallbackHandler`

Reply:
Renamed to `setPlatformNavigationCallbackDelegate` to match the rename of the `NavigationCallbackDelegate` class to `PlatformNavigationCallbackDelegate`.